### PR TITLE
Update 11-16-21-alueker-project-update-and-schedule

### PR DESCRIPTION
**Pull request by alueker**
Changed author name to correct name, assignment should appear on website after the edit

### CODE REVIEW

File: _posts/alueker/11-16-21-alueker-project-update-and-schedule [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-author: alueker
+author: Andrew Lueker
```
Comment (MishaCienfuegos) [inline _posts/alueker/11-16-21-alueker-project-update-and-schedule]: For your post to show up under your name on the People page, you will want the author to match your alias in the `config.yml` [file](https://github.com/inf380p/inf380p.github.io/blob/production/_config.yml#L107). It looks to me like they match up fine before this change. Does that seem true?

**Review by MishaCienfuegos — CHANGES_REQUESTED** [review]
It looks to me like there are a few changes that should be made to the file title so that your post displays properly on the class website. Check out the structure of the titles of [your other posts](https://github.com/inf380p/inf380p.github.io/tree/production/_posts/alueker), particularly the date and file extension.

File: _posts/alueker/11-16-21-alueker-project-update-and-schedule (hunk 1 above)
Comment (alueker) [reply _posts/alueker/11-16-21-alueker-project-update-and-schedule]: I believe I changed my alias after the first four assignments were uploaded. Double check it but I changed my name Andrew Lueker and I didn't see the project proposal or project update get uploaded to the site.

Comment (MishaCienfuegos) [reply _posts/alueker/11-16-21-alueker-project-update-and-schedule]: On line 107 of the [config.yml](https://github.com/inf380p/inf380p.github.io/blob/production/_config.yml#L107) it looks like your alias is still `alueker`. Did you see my note above about the file name, especially the date and file extension? That will also mess with how and where your post shows up on the website.

If you want to test the change as is,  we can merge and see if that resolves the problem of your November 16th update not showing up. But I still recommend those changes to the file name!

**Review by alueker — COMMENTED** [review]
This is more concise

File: _posts/alueker/11-16-21-alueker-project-update-and-schedule (hunk 1 above)
Comment (alueker) [reply _posts/alueker/11-16-21-alueker-project-update-and-schedule]: I see what you mean, I think you're right, I believe the issue was not adding the .md at the end of the file name

Comment (MishaCienfuegos) [reply _posts/alueker/11-16-21-alueker-project-update-and-schedule]: Great, that should help format your post as markdown!

You may also want to change the date format in the file name to `YYYY-MM-DD-` format. That specific syntax is important to the code that builds the website; without it, your post won't show up in the expected places.

Comment (alueker) [reply _posts/alueker/11-16-21-alueker-project-update-and-schedule]: Okay, date changed, everything should be good to go now!

File: _posts/alueker/2021-11-16-alueker-project-update-and-schedule.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ In my previous post I made a bold claim about the schedule of my game, a fully i
...
 In order to program the game into Python, there's a series of tasks that need to be completed for the project to be successful:
 
-- Align button commands (i.e. space bar for Bop, up key for Spin, down for Pull, etc.) with in-game commands
-- Create a timer that gradually degrades (gets faster as game progresses)
-- Create a score to count how many correct commands were performed in a row 
-- Create a score max that leads to a win screen
+1 Align button commands (i.e. space bar for Bop, up key for Spin, down for Pull, etc.) with in-game commands
+2 Create a timer that gradually degrades (gets faster as game progresses)
+3 Create a score to count how many correct commands were performed in a row 
+4 Create a score max that leads to a win screen
```
Comment (eah13) [inline _posts/alueker/2021-11-16-alueker-project-update-and-schedule.md]: For an ordered list in markdown, you'll want a `.` after the number:

1 this is just the numeral

1. This has a trailing period.

**Comment by eah13** [comment]
@alueker nice work sticking with these changes. One final suggestion is to add periods so you get a nice list on your post. Other than that I think this is ready to go.

File: _posts/alueker/2021-11-16-alueker-project-update-and-schedule.md (hunk 2 above)
Comment (alueker) [reply _posts/alueker/2021-11-16-alueker-project-update-and-schedule.md]: Okay, fixed the periods, should be good to launch now.

**Review by MishaCienfuegos — APPROVED** [review]
Thanks for sticking with us through these edits. I think this looks great!

**Pull request merged**